src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -28,15 +28,15 @@ alt: mobile device with app panel open
   </figcaption>
 </figure>
 
-As the capability gap between web and native gets smaller, it gets easier to
-offer the same experience for both web and native users. This may lead to cases
-where users have both web and native versions of the same app installed on the
-same device. Apps should be able to detect this situation.
+The [`getInstalledRelatedApps()`][spec] makes it possible for a page to check if
```

Comment:
`spec` should probably point to https://wicg.github.io/get-installed-related-apps/spec/ now rather than the github repo

Reply:
Updated